product-cli/cmd/cluster/aws/create.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"context"
+	"github.com/spf13/cobra"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1beta1"
+	hypershiftaws "github.com/openshift/hypershift/cmd/cluster/aws"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "aws",
+		Short:        "Creates basic functional HostedCluster resources on AWS",
+		SilenceUsage: true,
+	}
+
+	opts.AWSPlatform = core.AWSPlatformOptions{
+		AWSCredentialsFile: "",
+		Region:             "us-east-1",
+		InstanceType:       "",
+		RootVolumeType:     "gp3",
+		RootVolumeSize:     120,
+		RootVolumeIOPS:     0,
+		EndpointAccess:     string(hyperv1.Public),
+	}
+
+	cmd.Flags().StringVar(&opts.AWSPlatform.AWSCredentialsFile, "aws-creds", opts.AWSPlatform.AWSCredentialsFile, "Filepath to an AWS credentials file.")
+	cmd.Flags().StringVar(&opts.AWSPlatform.IAMJSON, "iam-json", opts.AWSPlatform.IAMJSON, "Filepath to a file containing IAM information for the cluster; if this is not specified, IAM will be automatically created.")
```

Comment:
is the iam-json use case needed for a known use case atm?

Reply:
Flag removed